partials/singles/single-post-fm.php

```diff
@@ -0,0 +1,48 @@
+<?php
+  $meta = get_post_meta($post->ID);
+?>
+
+<div class="row">
+  <div class="col col24">
+    <h4>FM</h4>
+  </div>
+</div>
+
+<div class="row">
+  <div class="col col12">
+    <h1 class="js-fix-widows"><?php the_title(); ?></h1>
+    // social sharing
+    <?php
+      if (!empty($meta['_cmb_dl_mp3'])) {
+        echo '<a href="' . $meta['_cmb_dl_mp3'][0] . '">Download mp3</a>';
+      }
+    ?>
+    <p>Published <?php the_time('jS F Y'); ?></p>
+  </div>
+
+  <div class="col col12">
+    <?php the_content(); ?>
+  </div>
+</div>
+
+<div class="row">
+  <div class="col col24">
+    <?php
+      if (!empty($meta['_cmb_sc'][0])) {
+    ?>
+        <iframe src="https://w.soundcloud.com/player/?url=<?php echo urlencode($meta['_cmb_sc'][0]); ?>" width="100%" height="200" scrolling="no" frameborder="no"></iframe>
+
+      <?php
+        if ($meta['_cmb_is_resonance'][0]) {
```

Comment:
maybe in single is fine like that
is there another youtube related functionality in the single beyond embedding the vid?